inference-engine/tests/functional/plugin/shared/src/single_layer_tests/eltwise.cpp

```diff
@@ -7,149 +7,137 @@
 
 #include <map>
 
-#include "common_test_utils/common_layers_params.hpp"
-#include "common_test_utils/common_utils.hpp"
-#include "common_test_utils/test_common.hpp"
-#include "common_test_utils/test_constants.hpp"
-#include "common_test_utils/xml_net_builder/ir_net.hpp"
-#include "common_test_utils/xml_net_builder/xml_filler.hpp"
 #include "functional_test_utils/layer_test_utils.hpp"
 #include "ngraph_functions/builders.hpp"
 #include "ngraph_functions/utils/ngraph_helpers.hpp"
-#include "ie_core.hpp"
+
 #include "single_layer_tests/eltwise.hpp"
 
-using namespace EltwiseTestNamespace;
+namespace LayerTestsDefinitions {
+std::ostream& operator<<(std::ostream & os, EltwiseParams::InputLayerType type) {
+    switch (type) {
+        case EltwiseParams::InputLayerType::CONSTANT:
+            os << "CONSTANT";
+            break;
+        case EltwiseParams::InputLayerType::PARAMETER:
+            os << "PARAMETER";
+            break;
+        default:
+            THROW_IE_EXCEPTION << "NOT_SUPPORTED_INPUT_LAYER_TYPE";
+    }
+    return os;
+}
 
-std::string EltwiseLayerTest::getTestCaseName(testing::TestParamInfo<eltwiseLayerTestParamsSet> obj) {
-    EltwiseOpType op;
-    ParameterInputIdx primary_input_idx;
-    InputLayerType secondary_input_type;
-    InferenceEngine::Precision prec;
-    InferenceEngine::SizeVector vec;
-    LayerTestsUtils::TargetDevice dev;
+std::ostream& operator<<(std::ostream & os, EltwiseParams::OpType type) {
```

Comment:
I recommend moving these operators implementation to inference-engine/tests/ngraph_functions/include/ngraph_functions/utils/ngraph_helpers.cpp